.github/fabricbot.json

```diff
@@ -482,275 +482,7776 @@
+                  "action": "closed"
+                }
               },
               {
-                "operator": "not",
-                "operands": [
-                  {
-                    "name": "hasLabel",
-                    "parameters": {
-                      "label": "area-System.Memory"
-                    }
-                  }
-                ]
+                "name": "addedToMilestone",
+                "parameters": {}
               }
             ]
           },
           {
-            "name": "isAction",
+            "name": "hasLabel",
             "parameters": {
-              "action": "unlabeled"
+              "label": "untriaged"
             }
-          },
+          }
+        ]
+      }
+    }
+  },
+  {
+    "taskSource": "fabricbot-config",
+    "taskType": "trigger",
+    "capabilityId": "IssueResponder",
+    "subCapability": "PullRequestResponder",
+    "version": "1.0",
+    "config": {
+      "taskName": "Assign Team PRs to author",
+      "actions": [
+        {
+          "name": "assignToUser",
+          "parameters": {
+            "user": {
+              "type": "prAuthor"
+            }
+          }
+        }
+      ],
+      "eventType": "pull_request",
+      "eventNames": [
+        "pull_request"
+      ],
+      "conditions": {
+        "operator": "and",
+        "operands": [
           {
-            "name": "isInProject",
+            "name": "isAction",
             "parameters": {
-              "projectName": "Area Pod: Adam / David - Issue Triage",
-              "isOrgProject": true
+              "action": "opened"
             }
+          },
+          {
+            "operator": "not",
+            "operands": [
+              {
+                "name": "activitySenderHasPermissions",
+                "parameters": {
+                  "permissions": "read"
+                }
+              }
+            ]
           }
         ]
-      },
-      "eventType": "issue",
-      "eventNames": [
-        "issues",
-        "project_card"
-      ],
-      "taskName": "[Area Pod: Adam / David - Issue Triage] Moved to Another Area",
+      }
+    }
+  },
+  {
+    "taskSource": "fabricbot-config",
+    "taskType": "trigger",
+    "capabilityId": "IssueResponder",
+    "subCapability": "IssuesOnlyResponder",
+    "version": "1.0",
+    "config": {
+      "taskName": "Needs-author-action notification",
       "actions": [
         {
-          "name": "addToProject",
+          "name": "addReply",
           "parameters": {
-            "projectName": "Area Pod: Adam / David - Issue Triage",
-            "columnName": "Triaged",
-            "isOrgProject": true
+            "comment": "This issue has been marked `needs-author-action` since it may be missing important information. Please refer to our [contribution guidelines](https://github.com/dotnet/runtime/blob/main/CONTRIBUTING.md#writing-a-good-bug-report) for tips on how to report issues effectively."
```

Comment:
```suggestion
            "comment": "This issue has been marked `needs-author-action` and may be missing some important information."
```

Reply:
Good catch; thanks. I'll update the scripts and regenerate to update that.